platform-server-rest/src/main/java/uno/cod/platform/server/rest/RestUrls.java

```diff
@@ -69,5 +69,8 @@
     public static final String TEAMS_CANONICAL_NAME_INVITATION_DECLINE = "/teams/{canonicalName}/invitation/decline";
     public static final String TEAMS_CANONICAL_NAME_INVITATIONS = "/teams/{canonicalName}/invitations";
 
+    public static final String LOCATIONS = "/locations";
+    public static final String LOCATIONS_CHALLENGE = "/locations/challenge/{id}";
```

Comment:
please use `canonicalName` and respect the RestUrls format `LOCATIONS_CHALLENGE _CANONICAL_NAME`